app/src/main/java/com/eveningoutpost/dexdrip/Models/BgReading.java

```diff
@@ -928,6 +940,7 @@ public String toJSON() {
             jsonObject.put("raw_calculated", raw_calculated);
             jsonObject.put("raw_data", raw_data);
             jsonObject.put("calculated_value_slope", calculated_value_slope);
+            jsonObject.put("calibration_uuid", calibration_uuid);
```

Comment:
Looking at a,b,c I think that they are used on the fllower. For example on the function:
displayCurrentInfo()
that calls displayCurrentInfoFromReading().
That calls estimated_bg().

There are many other calls that seems to be using a,b,c.
I'm not sure that the extra data is a real issue except for when sending data for the entire day, which should not really happen (but I guess that data costs differently in other places).

If we do want to save this data, than my suggestion is this:
Since a,b,c are double, they typically looks like:
a = 2.67846736149458e-10, b = -772.056833308984, c = 556355252287477.0
I believe that keeping more than 5 digits has no real influence on our accuracy. 
So we can probably send:
a = 2.6784, b = -772.05, c=55635e10
This will have almost the same values, but will use a much smaller space size.
I haven't still looked at how this look at json, but I wanted to know what do you think.

Thanks
Tzachi